src/nodes/materials/NodeMaterial.js

```diff
@@ -41,7 +41,7 @@ class NodeMaterial extends Material {
 		this.forceSinglePass = false;
 
 		this.fog = true;
-		this.lights = true;
+		this.lights = false;
```

Comment:
Probably better to set `lights` to `false` by default since we should not assume all custom node materials implement proper lighting models.

Reply:
I totally agree.

Reply:
Let's do that in a separate PR though. The testing for such changes can take a bit time.